src/org/isf/patient/manager/PatientBrowserManager.java

```diff
@@ -0,0 +1,402 @@
+package org.isf.patient.manager;
+
+import java.util.*;
```

Comment:
The project standards do not allow wildcard imports; you need to enumerate the imprts.